libcloud/dns/drivers/powerdns.py

```diff
@@ -248,18 +258,18 @@ def create_zone(self, domain, type=None, ttl=None, extra={}):
         payload.update(extra)
         zone_id = domain + '.'
         try:
-            self.connection.request(action=action, data=json.dumps(payload),
-                                    method='POST')
+            self.connection.request(action = action, data = json.dumps(payload),
+                                    method = 'POST')
         except BaseHTTPError:
             e = sys.exc_info()[1]
-            if e.code == httplib.UNPROCESSABLE_ENTITY and \
-               e.message.startswith("Domain '%s' already exists" % domain):
-                raise ZoneAlreadyExistsError(zone_id=zone_id, driver=self,
-                                             value=e.message)
+            if e.code == httplib.UNPROCESSABLE_ENTITY and\
+                e.message.startswith("Domain '%s' already exists" % domain):
+                raise ZoneAlreadyExistsError(zone_id = zone_id, driver = self,
+                                             value = e.message)
             raise e
-        return Zone(id=zone_id, domain=domain, type=None, ttl=None,
-                    driver=self, extra=extra)
-
+        return Zone(id = zone_id, domain = domain, type = None, ttl = None,
```

Comment:
Can you please revert all the style changes you have done? They don't comply with pep8 we follow and also make the pull request hard to review (hard to differentiate between actual code and style changes).

Thanks!